WebContent/WEB-INF/jsp/watchList.jsp

```diff
@@ -329,6 +329,7 @@
 
           if (owner) {
               show("p"+ pointId +"MoveUp");
+              show("p"+ pointId +"pointDetails");
```

Comment:
I would prefer there were no changes in the old UI because I cannot predict the consequences of such a change and it will be very difficult to improve the operation with the use of the dojo framework, different versions of jquery, and jquery plug-ins.

Reply:
The same situation - for automation tests.That is the unique name for points in watchlist to recongize points. That can be removed if you afraid about consequences, but from my side was checked.If we will have test server , there we should put this and we will make sure that this is not wrong line. For now , can be removed, but if you writing some frontend elements (vue) please share documentation when you creating some of elements for discussion.